torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -292,6 +302,47 @@ AOTITorchError aoti_torch__scaled_dot_product_flash_attention(
   });
 }
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch_convolution(
```

Comment:
Would it make sense to codegen these ?

Reply:
I think @eellison meant auto-generating shim functions. There were not many interfaces needed initially. Hopefully with more complete decomposition coverage, MM and Conv are the only cases we need to have in the C shim layer.